composer.json

```diff
@@ -17,8 +17,8 @@
         "psr/log": "^1.0"
     },
     "require-dev": {
-        "openeuropa/code-review": "^0.2",
-        "phpunit/phpunit": "^5||^6.0",
+        "openeuropa/code-review": "~0.2",
+        "phpunit/phpunit": "^5||^6",
```

Comment:
This notation with the two pipes is different than what is typically used in composer files. Normally a single pipe is sufficient. But it appears to work so I'm not rejecting it.